src/android/Provider.java

```diff
@@ -19,8 +19,8 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 package de.appplant.cordova.emailcomposer;
 
-import android.support.v4.content.FileProvider;
+import androidx.core.content.FileProvider;
```

Comment:
According to what I read, this is still valid
https://stackoverflow.com/questions/48534293/android-support-v4-content-fileprovider-not-found
But I would like to be sure. Do you confirm?

Reply:
Many other forks are using this change, such as

 - [11com7/cordova-plugin-email-composer](https://github.com/11com7/cordova-plugin-email-composer/blob/master/src/android/Provider.java#L22)
 - [CharityDynamics/cordova-plugin-email-composer](https://github.com/CharityDynamics/cordova-plugin-email-composer/blob/master/src/android/Provider.java#L22)

Reply:
this is indeed already a duplicate of PR #362